src/lib/es5.d.ts

```diff
@@ -207,13 +207,13 @@ interface ObjectConstructor {
      * Prevents the modification of existing property attributes and values, and prevents the addition of new properties.
      * @param o Object on which to lock the attributes.
      */
-    freeze<T extends {[idx: string]: U | null | undefined | object}, U extends string | bigint | number | boolean | symbol>(o: T): Readonly<T>;
+    freeze<const T extends {[idx: string]: U | null | undefined | object}, U extends string | bigint | number | boolean | symbol>(o: T): Readonly<T>;

```

Comment:
With the proposed change this overload could be removed altogether, right? 🤔

Reply:
I don't even know what this overload does.

Reply:
Something about literal types I'm sure, but I have no clue.

Reply:
IIRC this was added as “poor man’s const modifier” since that wasn’t available back then. It was added in: https://github.com/microsoft/TypeScript/pull/33767

Reply:
Yeah, I think this could be removed (I'm curious in all of the user tests and such with that change too, once we get results back for this one)